flink-libraries/flink-state-processing-api/src/test/java/org/apache/flink/state/api/input/MultiStateKeyIteratorTest.java

```diff
@@ -125,4 +185,117 @@ public void testIteratorRemovesFromAllDescriptors() throws Exception {
                             .count());
         }
     }
+
+    /** Test for lazy enumeration of inner iterators. */
+    @Test
+    public void testIteratorPullsSingleKeyFromAllDescriptors() throws AssertionError {
```

Comment:
I may missed something.
This UT may still not complete, right ?
Could you explain what we are verified from L197 - L202 ?